contrib/automation_tests/test_cases/top_down_tab.py

```diff
@@ -108,18 +108,39 @@ def _verify_rows_when_tree_collapsed(self, tree_view_table):
         self._verify_first_rows(tree_view_table, expectations)
         logging.info("Verified content of top-down view when all threads are collapsed")
 
-    def _verify_rows_when_thread_expanded(self, tree_view_table):
+    def _verify_rows_when_thread_and_errors_expanded(self, tree_view_table):
         logging.info("Expanding a thread of the top-down view")
         tree_view_table.get_item_at(1, 0).double_click_input()
         expected_first_thread_child_count = 2
-        self._verify_row_count(tree_view_table,
-                               self.EXPECTED_THREAD_COUNT + expected_first_thread_child_count)
+        expected_total_child_count = self.EXPECTED_THREAD_COUNT + expected_first_thread_child_count
+        self._verify_row_count(tree_view_table, expected_total_child_count)
+
+        logging.info("Expanding a thread of the top-down view")
+        tree_view_table.get_item_at(3, 0).double_click_input()
+        expected_second_thread_child_count = 1
+        expected_total_child_count += expected_second_thread_child_count
+        self._verify_row_count(tree_view_table, expected_total_child_count)
+
+        tree_view_table.get_item_at(4, 0).double_click_input()
+        expected_third_thread_child_count = 1
```

Comment:
Same here, you did not expand a thread, but the error node.

Reply:
Done.